docs/auth0_users_search.md

```diff
@@ -25,7 +25,7 @@ auth0 users search [flags]
 
 ```
       --json           Output in json format.
-  -n, --number int     Number of users, that match the search criteria, to retrieve. Maximum result number is 1000. If limit is hit, please refine the search query. (default 50)
+  -n, --number int     Number of users to retrieve. Minimum 1, maximum 1000. (default 50)
```

Comment:
Here the help text actually made sense as this is the only list/search command where you can pass a query. 
```suggestion
  -n, --number int     Number of users, that match the search criteria, to retrieve. Minimum 1, maximum 1000. If limit is hit, refine the search query. (default 50)
```